lib/html_formatter.js

```diff
@@ -80,6 +80,7 @@ module.exports = (function () {
             data: {
                 status: step.result ? step.result.status : '',
                 errorDetails: step.result ? toHtmlEntities(step.result.error_message) : '',
+                duration: step.result.duration ? (step.result.duration / 1000000000).toFixed(2) + 's' : 'no data',
```

Comment:
looknij tu: 
coverter :) http://www.convertunits.com/from/nanosecond/to/second
http://comments.gmane.org/gmane.comp.programming.tools.cucumber/16882
https://groups.google.com/forum/#!topic/cukes/itAKGVwJHFg